src/languages/marko.coffee

```diff
@@ -1,8 +1,15 @@
+# Get Atom defaults
+scope = ['text.marko']
+tabLength = atom?.config.get('editor.tabLength', scope: scope) ? 4
+softTabs = atom?.config.get('editor.softTabs', scope: scope) ? true
+defaultIndentSize = (if softTabs then tabLength else 4)
+defaultIndentChar = (if softTabs then " " else "\t")
+defaultIndentWithTabs = not softTabs
+
 module.exports = {
 
   name: "Marko"
   namespace: "marko"
-  fallback: ['html']
```

Comment:
Marko templates no longer use the HTML syntax (Marko uses a variation of HTML that is not always valid HTML). Therefore, you would not be able to use the default HTML beautifier to beautify a Marko v3 template. Also, currently the Marko beautifier doesn't support the majority of the settings that the HTML beautifier supports. That's my thought process for removing `html` as a fallback, but please let me know if I am mistaken. Thanks again!